src/display/annotation_storage.js

```diff
@@ -33,20 +33,19 @@ class AnnotationStorage {
 
   /**
    * Get the value for a given key if it exists
-   * or store and return the default value
+   * or return the default value
    *
    * @public
    * @memberof AnnotationStorage
    * @param {string} key
    * @param {Object} defaultValue
    * @returns {Object}
    */
-  getOrCreateValue(key, defaultValue) {
+  getOrDefaultValue(key, defaultValue) {
```

Comment:
Any reason to not simply naming this `getValue` instead now, since it would mirror the existing `setValue` method nicely?